docs/UsersGuide.asciidoc

```diff
@@ -1133,10 +1133,22 @@ how to configure the limits. Cleanup-related parameter for the REST API can
 be found in the <<UsersGuide.asciidoc#_asset_handling,Asset handling>> section under
 <<UsersGuide.asciidoc#_use_of_the_rest_api,Use of the REST API>>.
 
-==== Cleanup strategy ====
+==== Timers and triggers ====
+
+Cleanup can be triggered in different ways. One option is to use `minion_task_triggers`
+and specify tasks via `on_job_done`. Another way to do that is to use the systemd timers
+`openqa-enqueue-asset-cleanup` and `openqa-enqueue-result-cleanup` to periodically
+run those tasks. Both can be used separately or in combination.
+
+The relevant minion tasks are:
 
-openQA frequently checks whether assets need to be removed according to
-the configured limits.
+- limit_results_and_logs
```

Comment:
This is under "Asset cleanup" so listing the result cleanup here seems a bit odd. Maybe the whole sub-section should not be under "Asset cleanup". I'd also like to note that this list isn't complete as we also cleanup bugs and audit events and the result cleanup has extra tasks for screenshots. I would just say `limit_*` tasks.

Reply:
I decided to document other tasks and move the section out, since it should be useful to mention what tasks we actually have.